microsoft-edge/extensions-chromium/publish/contact-us.md

```diff
@@ -0,0 +1,33 @@
+---
+description: Contact support information for Microsoft Edge extension development.
+title: Contact Edge extensions support
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 01/19/2021
+ms.topic: support
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, support
+---
+
+# Contact Edge extension support
+
+## Contact us for any issues or feedback
+
+1. For any issues related to certification, please submit a [New Support Request][ExtensionsSupportForm].
```

Comment:
Suggest we make this a little more clear. 

If you have submitted an extension via the Partner Center and have a question related to its review/certification status, please submit a New Support Request.